stuffer/s2n_stuffer.c

```diff
@@ -104,7 +104,7 @@ int s2n_stuffer_resize(struct s2n_stuffer *stuffer, const uint32_t size)
 int s2n_stuffer_resize_if_empty(struct s2n_stuffer *stuffer, const uint32_t size)
 {
     if (stuffer->blob.data == NULL) {
-        GUARD(s2n_stuffer_resize(stuffer, size));
+        GUARD(s2n_realloc(&stuffer->blob, size));
```

Comment:
`s2n_stuffer_resize` performed a tainted check and a static check. Should those checks be added to to this function now that a direct realloc is done?

Reply:
Yes.